code/game/objects/items/devices/sensor_device.dm

```diff
@@ -5,7 +5,7 @@
 	icon_state = "scanner"
 	w_class = WEIGHT_CLASS_SMALL
 	slot_flags = ITEM_SLOT_BELT
-	custom_price = 100
+	custom_price = 1750
```

Comment:
If you're gonna jack up the price of handheld suit sensor monitors this much, could you please at least make medical doctors and the CMO spawn with one on their person at roundstart? This change plus the recent removal of the suit sensor monitoring console in Medbay on Metastation means that medical doctors (pretty much) can't monitor suit sensors near roundstart on Metastation without breaking into somewhere or acquiring cash.

Reply:
Oh, nevermind, they're still free for medical doctors. Carry on.